drivers/SmartThings/matter-switch/src/eve-energy/init.lua

```diff
@@ -217,13 +217,15 @@ end
 -------------------------------------------------------------------------------------
 
 local function watt_attr_handler(driver, device, ib, zb_rx)
+  print("watt_attr_handler")
```

Comment:
Did you mean to include this? If so would be better to switch to using logging

Reply:
Did not mean to include this! Still cleaning up this PR for testing purposes.